flink-cdc-common/src/main/java/com/ververica/cdc/common/event/OperationType.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.common.event;
+
+/** Enum {@code OperationType} describes the type of operation that the change event reports. */
+public enum OperationType {
+    INSERT,
+    UPDATE,
+    REPLACE,
+    DELETE,
+
+    CREATE,
+    RENAME,
```

Comment:
can we use `ALTER` type to replace this? As @PatrickRen discussed before, when adding new SchemaChangeEvent class, we can use `instance of` judgment rather than adding enum in OperationType.

Reply:
OperationType looks like describing event type for data change event instead of schema change event, we should introduce another type to describe the later one.